test/releases.test.js

```diff
@@ -0,0 +1,68 @@
+        ...baseConfig,
+        'change-title-escapes': '\\'
+      }
+      const changelog = generateChangeLog(pullRequests, config)
+      expect(changelog).toEqual('* A1 (#1) @ghost\n* B2 (#2) @ghost\n* Adds missing <example> (#3) @jetersen\n* `#code_block` (#4) @jetersen\n* Fixes #4 (#5) @Happypig375\n* 2*2 should equal to 4*1 (#6) @jetersen\n* Rename __confgs\\\\confg.yml to __configs\\\\config.yml (#7) @ghost\n* Adds @nullable annotations to the 1*1+2*4 test in `tests.java` (#0) @Happypig375')
+    })
+    it('escapes titles with \\<*_& correctly', () => {
+      const config = {
+        ...baseConfig,
+        'change-title-escapes': '\\<*_&'
+      }
+      const changelog = generateChangeLog(pullRequests, config)
+      expect(changelog).toEqual('* A1 (#1) @ghost\n* B2 (#2) @ghost\n* Adds missing \\<example> (#3) @jetersen\n* `#code_block` (#4) @jetersen\n* Fixes #4 (#5) @Happypig375\n* 2\\*2 should equal to 4\\*1 (#6) @jetersen\n* Rename \\_\\_confgs\\\\confg.yml to \\_\\_configs\\\\config.yml (#7) @ghost\n* Adds @nullable annotations to the 1\\*1+2\\*4 test in `tests.java` (#0) @Happypig375')
+    })
+    it('escapes titles with @s correctly', () => {
+      const config = {
+        ...baseConfig,
+        'change-title-escapes': '@'
+      }
+      const changelog = generateChangeLog(pullRequests, config)
+      expect(changelog).toEqual('* A1 (#1) @ghost\n* B2 (#2) @ghost\n* Adds missing <example> (#3) @jetersen\n* `#code_block` (#4) @jetersen\n* Fixes #4 (#5) @Happypig375\n* 2*2 should equal to 4*1 (#6) @jetersen\n* Rename __confgs\\confg.yml to __configs\\config.yml (#7) @ghost\n* Adds @<!---->nullable annotations to the 1*1+2*4 test in `tests.java` (#0) @Happypig375')
+    })
+    it('escapes titles with @s and #s correctly', () => {
+      const config = {
+        ...baseConfig,
+        'change-title-escapes': '@#'
+      }
+      const changelog = generateChangeLog(pullRequests, config)
+      expect(changelog).toEqual('* A1 (#1) @ghost\n* B2 (#2) @ghost\n* Adds missing <example> (#3) @jetersen\n* `#code_block` (#4) @jetersen\n* Fixes #<!---->4 (#5) @Happypig375\n* 2*2 should equal to 4*1 (#6) @jetersen\n* Rename __confgs\\confg.yml to __configs\\config.yml (#7) @ghost\n* Adds @<!---->nullable annotations to the 1*1+2*4 test in `tests.java` (#0) @Happypig375')
+    })
+    it('escapes titles with \\<@*_&`# correctly', () => {
+      const config = {
+        ...baseConfig,
+        'change-title-escapes': '\\<@*_&`#'
+      }
+      const changelog = generateChangeLog(pullRequests, config)
+      expect(changelog).toEqual('* A1 (#1) @ghost\n* B2 (#2) @ghost\n* Adds missing \\<example> (#3) @jetersen\n* \\`#<!---->code\\_block\\` (#4) @jetersen\n* Fixes #<!---->4 (#5) @Happypig375\n* 2\\*2 should equal to 4\\*1 (#6) @jetersen\n* Rename \\_\\_confgs\\\\confg.yml to \\_\\_configs\\\\config.yml (#7) @ghost\n* Adds @<!---->nullable annotations to the 1\\*1+2\\*4 test in \\`tests.java\\` (#0) @Happypig375')
```

Comment:
replace `toEqual` with [`toMatchInlineSnapshot`](https://jestjs.io/docs/en/expect#tomatchinlinesnapshotpropertymatchers-inlinesnapshot)

Reply:
Does `toMatchInlineSnapshot` allow indentation? There is
https://github.com/release-drafter/release-drafter/blob/842a9a8162bb2f1ce14de650121978c9d5a9c4d3/test/index.test.js#L187-L202
but this PR's tests seem to fail.

Reply:
Because it is inside a object :)

When you use `toMatchInlineSnapshot` on a string, it cannot tolerate it 😅.

Reply:
You can also use `toMatchSnapshot` which will create a file where indentation shouldn't be an issue.

Reply:
Will wrap it in an object

Reply:
```
    - Snapshot  - 13
    + Received  + 10

    -
    -         Object {
    -           "log": "
    -         * A1 (#1) @ghost
    -         * B2 (#2) @ghost
    -         * Adds missing \<example> (#3) @jetersen
    -         * \`\#code\_block\` (#4) @jetersen
    -         * Fixes #<!--->4 (#5) @Happypig375
    -         * 2\*2 should equal to 4\*1 (#6) @jetersen
    -         * Rename \_\_confgs\\confg.yml to \_\_configs\\config.yml (#7) @ghost
    -         * Adds @<!--->nullable annotations to the 1\*1+2\*4 test in `tests.java` (#0) @Happypig375
    -         ",
    -         }
    + Object {

```

Reply:
I'll throw away indentation.

Reply:
```
    - Snapshot  - 0
    + Received  + 0
```
???

Reply:
Trying to use `toMatchInlineSnapshot` is literally a waste of time.

Reply:
@jetersen Any suggestions?